pages/community/election.md

```diff
@@ -0,0 +1,82 @@
+
+The US-RSE Steering Committee sets its own meeting frequency and timing.  Currently, the Steering Committee is meeting every two weeks.  The Steering Committee meeting frequency and schedule for 2022 will be determined once new members are elected.  
+
+The responsibilities of Steering Committee members are detailed in the [US-RSE Membership and Governance](https://github.com/USRSE/documents/blob/master/governance.md) document. They include:
+
+* Setting the overall policy and direction of the US-RSE Association
+* Being responsible for day-to-day activities, operations, and development of the US-RSE Association
+
+Additionally, Steering Committee members are expected to:
+
+* Attend and participate in Steering Committee meetings
+* Follow and participate in Steering Committee discussions on Slack
+* Contribute to US-RSE operations through activities such as
+  * Updating the website
+  * Writing newsletters and communications
+  * Organizing and leading community calls and events
+  * Engaging with community members on Slack
+* Help further the community by
+  * Participating on international RSE committees
+  * Organizing BOFs, workshops, meetups, and other events at relevant conferences
+  * Leading US-RSE working groups
+  * Writing blog posts
+  * Bringing new ideas to the organization
+
+## Nominations and Candidate Information
+
+Individuals may nominate themself or another member.  In cases when a nomination is not a self-nomination, the Election Chair will contact the nominated individual to confirm their interest in participating.
+
+On the nomination form, the following information is required:
+
+**If Nominating Someone Else:**
+
+* Your name and email
+* Nominee’s name and email
+* Short statement about why you think the nominated individual will make a good Steering Committee member
+
+Nominations of others will only be shared with the Election Chairs and the nominated individual.  The information will not be made public.
+
+**Self-nomination or Accepted Nomination:**
+
+* Your name and a publicly-shareable email
+* A link to a resume, CV, LinkedIn profile, or professional website
+* Why do you want to be on the US-RSE Steering Committee? (<=1000 characters)
+* How do you contribute to the US-RSE Association or RSE community more broadly? (<= 1500 characters)
+
+All candidate information will be shared on the US-RSE website.  Additionally, candidates will have the opportunity to share a short statement (live or recorded) during the US-RSE General meeting in December.  
```

Comment:
I think these should all be recorded - having both formats is a bit awkward and perhaps also unfair.

Reply:
I agree. A recording with a time limit seems the most fair and controlled. But having candidates available live for questions/discussion should be encouraged

Reply:
I also agree that we should have everyone in the same format. So a recording followed by live Q&A is a good idea.